.meshrc.yaml

```diff
@@ -39,6 +39,13 @@ sources:
             responseTypeName: BulkDownload
             headers:
               Cookie: "{context.headers['cookie']}"
+          - type: Query
+            field: BulkDownloadCSV
```

Comment:
Would it be more desirable to have separate graphQL calls for different download types (CG overview and metadata)?  They can call common functions in the front / back end, but I feel like it's more explicit.  The current bulk download endpoint has a lot of branching logic, and I feel like it'd be nice to move away from that...

Reply:
Without Sample Metadata we don't have to worry about this now right? I can change the name to something more focused - `BulkDownloadOverview`?